components/script/dom/workletglobalscope.rs

```diff
@@ -93,6 +94,7 @@ impl WorkletGlobalScope {
         self.globalscope.evaluate_js_on_global_with_result(
             &*script,
             rval.handle_mut(),
+            ScriptFetchOptions::default_classic_script(&self.globalscope),
```

Comment:
Please file a followup issue to take a ScriptFetchOptions argument so we can make it match https://github.com/servo/servo/blob/854cc793861c31c395387a1197c40e840bc95826/components/script/dom/worklet.rs#L627-L634.

Reply:
Thanks for the reminder! I just filed it at #27308!